KomaMRICore/src/rawdata/ISMRMRD.jl

```diff
@@ -85,19 +85,24 @@ function signal_to_raw_data(
     mink = minimum(ktraj, dims=1)
     maxk = maximum(ktraj, dims=1)
     Wk = maxk .- mink
-    Δx = 1 ./ Wk[1:2] #[m] Only x-y
+    Δx = 1 ./ Wk[1:3] #[m] Only x-y
     Nx = get(seq.DEF, "Nx", 1)
     Ny = get(seq.DEF, "Ny", 1)
     Nz = get(seq.DEF, "Nz", 1)
+    Nd_seq = (Nx > 1) + (Ny > 1) + (Nz > 1)
+    if ndims != Nd_seq; @warn("Seqfile is $Nd_seq dimensional but recon is $ndims."); end
     if haskey(seq.DEF, "FOV")
-        FOVx, FOVy, _ = seq.DEF["FOV"] #[m]
+        FOVx, FOVy, FOVz = seq.DEF["FOV"] #[m]
         if FOVx > 1 FOVx *= 1e-3 end #mm to m, older versions of Pulseq saved FOV in mm
         if FOVy > 1 FOVy *= 1e-3 end #mm to m, older versions of Pulseq saved FOV in mm
+        if FOVz > 1 FOVz *= 1e-3 end #mm to m, older versions of Pulseq saved FOV in mm
         Nx = round(Int64, FOVx / Δx[1])
         Ny = round(Int64, FOVy / Δx[2])
+        Nz = round(Int64, FOVy / Δx[3])
```

Comment:
The Ci gives an error saying that `NadcsPerImage = floor(Int, Nadcs / Nz)` => `Nz = 0`. This line could be the problem.
```suggestion
        Nz = round(Int64, FOVz / Δx[3])
```
I wonder if `Δx[3]` is `Inf`.